code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -274,13 +277,41 @@
 	attack_verb = list()
 	icon_state = initial(icon_state)
 
+/obj/item/melee/energy/sword/perform_technique(mob/living/carbon/human/target, mob/living/carbon/human/user, target_zone)
+	. = ..()
+	var/armor_reduction = target.get_blocked_ratio(target_zone, DAMAGE_BRUTE, DAMAGE_FLAG_EDGE|DAMAGE_FLAG_SHARP, damage = force)*100
+	var/obj/item/organ/external/affecting = target.get_organ(target_zone)
+	if(!affecting)
+		return
+	user.do_attack_animation(target)
+
+	if(target_zone == BP_HEAD || target_zone == BP_EYES || target_zone == BP_MOUTH)
+		if(prob(70 - armor_reduction))
+			target.eye_blurry += 5
+			target.confused += 10
+			return TRUE
+
+	if(target_zone == BP_R_ARM || target_zone == BP_L_ARM || target_zone == BP_R_HAND || target_zone == BP_L_HAND)
+		if(prob(80 - armor_reduction))
+			if(target_zone == BP_R_ARM || target_zone == BP_R_HAND)
+				target.drop_r_hand()
+			else
+				target.drop_l_hand()
+			return TRUE
```

Comment:
This seems to allows to spamattack someone so that he would not be able to fight back at all, I think the chance of performing the drop is too high especially in consideration of unathi that can basically teleport to you to perform this

Reply:
This is exactly the same code as for normal swords, and I haven't noticed any major balance issues coming from it on the rare occasion that swordsmanship is used

Reply:
Normal swords attack delay is higher than energy one, no?

Reply:
> Normal swords attack delay is higher than energy one, no?

I don't think so